rpc/src/response.rs

```diff
@@ -69,7 +69,6 @@ impl<R> Wrapper<R> {
         }
     }
 
-    #[cfg(test)]
```

Comment:
IIRC this method was only ever supposed to be used in tests. Why the change?

Reply:
I wanted to use this type externally and there's no other way to construct it afaik. I can revert this and just make my own `Wrapper` type if you'd prefer